### ISSUE TITLE
Fatal Python error: PyThreadState_Get: no current thread

### ISSUE THREAD

**Issue by Doweig** [greenaddress/garecovery#24]
Hello there, I hope you are having a great day.

So I can't get it to be running on my machine, looking on the internet it looks like a python version mismatch, like the python version used to compiled is different than the one executing, but that's not the case. I check every step of the way I'm always using 2.7, stil can't get it too work. Any idea on what to do from there?

Many thanks in advance :D

```
$> virtualenv -p /usr/local/opt/python@2/bin/python2.7 venv
Already using interpreter /usr/local/opt/python@2/bin/python2.7
New python executable in /Users/doweig/dev/bitcoin_playground/garecovery/venv/bin/python2.7
Not overwriting existing python script /Users/doweig/dev/bitcoin_playground/garecovery/venv/bin/python (you must use /Users/doweig/dev/bitcoin_playground/garecovery/venv/bin/python2.7)
Installing setuptools, pip, wheel...done.
$> source venv/bin/activate
$> pip2.7 install --require-hashes -r tools/requirements.txt
Ignoring wallycore: markers '"linux" in sys_platform and python_version == "2.7"' don't match your environment
Ignoring wallycore: markers '"linux" in sys_platform and python_version == "3.5"' don't match your environment
Ignoring wallycore: markers '"linux" in sys_platform and python_version == "3.6"' don't match your environment
Ignoring wallycore: markers 'sys_platform == "darwin" and python_version == "3.6"' don't match your environment
Ignoring wallycore: markers 'sys_platform == "win32" and python_version == "3.6"' don't match your environment
Collecting https://github.com/ElementsProject/libwally-core/archive/release_0.6.1.tar.gz; (from -r tools/requirements.txt (line 11))
  Downloading https://github.com/ElementsProject/libwally-core/archive/release_0.6.1.tar.gz; (774kB)
    100% |████████████████████████████████| 778kB 2.2MB/s
Requirement already satisfied (use --upgrade to upgrade): wallycore==0.6.1 from https://github.com/ElementsProject/libwally-core/archive/release_0.6.1.tar.gz; in ./venv/lib/python2.7/site-packages (from -r tools/requirements.txt (line 11))
Requirement already satisfied: argcomplete==1.8.2 in ./venv/lib/python2.7/site-packages (from -r tools/requirements.txt (line 1)) (1.8.2)
Requirement already satisfied: pycoin==0.80 in ./venv/lib/python2.7/site-packages (from -r tools/requirements.txt (line 2)) (0.80)
Requirement already satisfied: python-bitcoinrpc==1.0 in ./venv/lib/python2.7/site-packages (from -r tools/requirements.txt (line 3)) (1.0)
Requirement already satisfied: wallycore==0.6.1 from https://github.com/ElementsProject/libwally-core/releases/download/release_0.6.1/wallycore-0.6.1-cp27-cp27m-macosx_10_12_x86_64.whl in ./venv/lib/python2.7/site-packages (from -r tools/requirements.txt (line 8)) (0.6.1)
$> pip2.7 install .
Processing /Users/doweig/dev/bitcoin_playground/garecovery
Building wheels for collected packages: greenaddress-recovery
  Running setup.py bdist_wheel for greenaddress-recovery ... done
  Stored in directory: /private/var/folders/yz/cy2h3y_d0qv2y0kl5vpngdk80000gn/T/pip-ephem-wheel-cache-_aVD1f/wheels/8a/a0/61/6e7596c9be1b9ad7bf295fa7f441f0df37ba72990dde8d3f35
Successfully built greenaddress-recovery
Installing collected packages: greenaddress-recovery
  Found existing installation: greenaddress-recovery 0.1
    Uninstalling greenaddress-recovery-0.1:
      Successfully uninstalled greenaddress-recovery-0.1
Successfully installed greenaddress-recovery-0.1
$> garecovery-cli 2of2 --nlocktime-file ~/Downloads/nlocktimes.zip -o garecovery.csv
Fatal Python error: PyThreadState_Get: no current thread
[1]    30482 abort      garecovery-cli 2of2 --nlocktime-file ~/Downloads/nlocktimes.zip -o
$> which garecovery-cli
/Users/doweig/dev/bitcoin_playground/garecovery/venv/bin/garecovery-cli
$> cat /Users/doweig/dev/bitcoin_playground/garecovery/venv/bin/garecovery-cli
#!/Users/doweig/dev/bitcoin_playground/garecovery/venv/bin/python2.7
# PYTHON_ARGCOMPLETE_OK

import sys
from garecovery import recoverycli

sys.exit(recoverycli.main(sys.argv))
```

**Comment by jgriffiths** [comment]
Hi,
Are you on a Mac machine? The pre-built wally binaries are only available for python 3.6 on mac, it seems that the 2.7 wally built from source is not working.

Please nuke the venv and try with python 3.6.

**Comment by jgriffiths** [comment]
No response and we now have wally binary installs which should fix binary build incompatibilities, closing.
